library/hawkular_alerts_group_trigger.py

```diff
@@ -122,37 +122,85 @@ def delete_group_trigger(self, group_id):
         except Exception as e:
             self.module.fail_json(msg="Failed to delete group trigger. Error: {error}".format(error=e))
 
-    def create_group_trigger(self, name, group_id, severity, enabled):
-        """ Creates a group trigger in Hawkular Alerting component
+    def required_updates(self, trigger, name, severity, enabled):
+        """ Checks whether an update is required for the group trigger
+
+            Returns:
+                True if the passed trigger name, severity or enabled status
+                are different from the ones set in the trigger, False otherwise
+        """
+        updates = {}
+        if (name and trigger.name != name):
+            updates["name"] = name
+        if (severity and trigger.severity != severity):
+            updates["severity"] = severity
+        if (enabled and trigger.enabled != enabled):
+            updates["enabled"] = enabled
+        return updates
+
+    def update_group_trigger(self, trigger, group_id, updates):
```

Comment:
I believe you could drop `group_id` param, fetch `trigger.id` where needed.
Not sure why it sounds better to me, your call.